ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-GAR3CP-nonprod.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseStateID": "LISTING_FOR_A_HEARING",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "hearing-schedule-access"
```

Comment:
I believe systemupdate userrole is missing. could you verify please?

Reply:
systemupdate userrole is in AuthorisationCaseStateGAspec.json